.github/workflows/dependency-review-bypass.yaml

```diff
@@ -0,0 +1,21 @@
+name: Dependency Review
+
+on:
+  pull_request:
+    paths-ignore:
```

Comment:
"When all the path names match patterns in paths-ignore, the workflow will not run. If any path names do not match patterns in paths-ignore, even if some path names match the patterns, the workflow will run."

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

I think this means this bypass will pretty much always still run, since other changes are typically included whenever dependencies are updated